run/local-cluster.sh

```diff
@@ -95,23 +96,29 @@ echo "* ${DNSMASQ_NAME}"
 # Local ports.
 PORT=8080
 
+# Start a data volume node for shared data.
+CERTS_DIR="/certs"
+CERTS_NAME="cockroach_certs"
+# Generate certs.
+docker run -v ${CERTS_DIR} --name=${CERTS_NAME} ${COCKROACH_IMAGE} create-ca-cert --certs=${CERTS_DIR} 2> /dev/null
+docker run --volumes-from=${CERTS_NAME} ${COCKROACH_IMAGE} create-node-cert --certs=${CERTS_DIR} localhost 127.0.0.1 2> /dev/null
+
 # Start all nodes.
 for i in $(seq 1 $NODES); do
   HOSTS[$i]="$COCKROACH_NAME$i.local"
   VOL="/data$i"
 
   # Command args specify two data directories per instance to simulate two physical devices.
-  START_ARGS="-gossip=${HOSTS[1]}:$PORT -stores=ssd=$VOL -addr=${HOSTS[$i]}:$PORT"
+  START_ARGS="-gossip=${HOSTS[1]}:$PORT -stores=ssd=$VOL -addr=${HOSTS[$i]}:$PORT -certs=${CERTS_DIR}"
   # Log (almost) everything.
   #START_ARGS="${START_ARGS} -v 7"
   # Node-specific arguments for node container.
-  NODE_ARGS="--hostname=${HOSTS[$i]} --name=${HOSTS[$i]} --dns=$DNS_IP"
+  NODE_ARGS="--hostname=${HOSTS[$i]} --volumes-from=${CERTS_NAME} --name=${HOSTS[$i]} --dns=$DNS_IP"
 
   # If this is the first node, initialize the cluster first.
   if [[ $i == 1 ]]; then
-      docker run -v $VOL $COCKROACH_IMAGE init $VOL 2> /dev/null
-      INIT_CID=$(docker ps -q -n 1)
-      NODE_ARGS="${NODE_ARGS} --volumes-from=$INIT_CID"
+      docker run -v $VOL --volumes-from=${CERTS_NAME} --name=cockroach_init $COCKROACH_IMAGE init -certs=${CERTS_DIR} $VOL 2> /dev/null
+      NODE_ARGS="${NODE_ARGS} --volumes-from=cockroach_init"
       # TODO(tobias): do we need to figure out how to clean up this volume?
```

Comment:
while you're at it, I think removing all containers in this test with the additional '-v' flag should clean up the volumes too.